api/src/services/monitoring.js

```diff
@@ -179,7 +180,20 @@ const getReplicationLimitLog = () => {
     });
 };
 
-const json = () => {
+const getConnectedUserLogs = (daysAgo) => {
+  const earliestTimestamp = moment().subtract(daysAgo, 'days').valueOf();
+  return db.medicLogs
+    .query(`logs/connected_users?startkey=${earliestTimestamp}`)
```

Comment:
`db.medicLogs` is a PouchDB object, so you can use the [rich API](https://pouchdb.com/api.html#query_database) to make this query instead of cobbling together a request path.

I also like to explicitly use `reduce:true` because although it's the default it's easier to work out what's going on if it's in there.

```suggestion
    .query('logs/connected_users', { startkey: earliestTimestamp, reduce: true })
```